src/labone/mock/automatic_server.py

```diff
@@ -226,59 +226,95 @@ async def get_with_expression(
         """
         return [await self.get(p) for p in await self.list_nodes(path=path_expression)]
 
-    async def set(self, value: AnnotatedValue) -> AnnotatedValue:
+    @t.overload
+    async def set(self, value: AnnotatedValue) -> AnnotatedValue: ...
+
+    @t.overload
+    async def set(
+        self,
+        value: Value,
+        path: LabOneNodePath,
+    ) -> AnnotatedValue: ...
+
+    async def set(self, value: AnnotatedValue | Value, path="") -> AnnotatedValue:
```

Comment:
Should we raise an Exception if `value` is `AnnotatedValue` and `path` is given.

Or another options is to document that whenever `AnnotatedValue` is used, path value is ignored.

Reply:
Isn't this documented by the overloaded function declarations?

Reply:
I would say yes, however passing both is not is not acknowledged in the docstring. However I think its fine.